components/telegram_bot_api/telegram_bot_api.app.js

```diff
@@ -89,5 +271,386 @@ module.exports = {
       };
       return await axios(config);
     },
+    /**
+     * Send a text message
+     *
+     * @param {String} chatId - Unique identifier for the target chat or
+     * username of the target channel (in the format `@channelusername`)
+     * @param {String} text - Text of the message to be sent, 1-4096 characters
+     * after entities parsing
+     * @param {Object} [opts] - An object containing additional configuration
+     * options for this method, as defined in [the API
+     * docs](https://core.telegram.org/bots/api#sendmessage)
+     * @returns The sent Message
+     */
+    async sendMessage(chatId, text, opts) {
+      return await this.sdk().sendMessage(chatId, text, opts);
+    },
+    /**
+     * Edit a text message
+     *
+     * One of chat_id, message_id, or inline_message_id must be provided in the
+     * `opts` parameter
+     *
+     * @param {String} text - New text of the message
+     * @param {Object} [opts] - An object containing additional configuration
+     * options for this method
+     * @param {Number|String} [opts.chatId] - Required if `inline_message_id` is
+     * not specified. Unique identifier for the target chat or username of the
+     * target channel (in the format @channelusername)
+     * @param {Number|String} [opts.messageId] - Required if inline_message_id
+     * is not specified. Identifier of the message to edit
+     * @param {Number|String} [opts.inlineMessageId] - Required if chat_id and
+     * message_id are not specified. Identifier of the inline message
+     * @param {...*} [opts.extraOpts] - Additional Telegram query options to be
+     * fed to the Telegram Bot API call, as defined in [the API
+     * docs](https://core.telegram.org/bots/api#editmessagetext)
+     * @returns The edited Message
+     */
+    async editMessageText(text, opts) {
+      const {
+        chatId,
+        messageId,
+        inlineMessageId,
+        ...extraOpts
+      } = opts;
+      if (!(chatId && messageId) && !inlineMessageId) {
+        throw new Error("chatId, messageId, or inlineMessageId is required");
+      }
```

Comment:
Maybe you should change the error message here since it seems like `chatId && messageId` and `inlineMessageId` are mutually exclusive
Also to make it more readable I would set a variable like `hasChatIdAndMessageId`
```suggestion
      const hasChatIdAndMessageId = chatId && messageId;
      if (!hasChatIdAndMessageId && !inlineMessageId) {
         throw new Error("chatId, messageId, or inlineMessageId is required");
      }
```